docs/architecture/deep-dives/database-replicas.md

```diff
@@ -0,0 +1,41 @@
+# Read-Only Database Replicas
+
+## Context
+
+Bitwarden Utilizes
+[Azure SQL Hyperscale](https://learn.microsoft.com/en-us/azure/azure-sql/database/service-tier-hyperscale?view=azuresql)
+
+Bitwarden utilizes the
```

Comment:
```suggestion
and its
```

Combine this with the line above it. I didn't initially notices the statement floating above this one.